src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -211,7 +231,7 @@ private IReadOnlyList<Uri> ParseServerUrlsImpl(ConfigurationKeyValue kv)
 			{
 				_logger?.Warning()
 					?.Log(nameof(EnvVarNames.ServerUrls)
-						+ " configuration option contains more than one URL which is not supported by the agent yet"
```

Comment:
Should we print something like this additionally here?

> `ServerUrls` config is obsolete, use `ServerUrl` instead.

Reply:
++ will add